doc/build-unix.md

```diff
@@ -54,8 +54,11 @@ System requirements
 --------------------
 
 C++ compilers are memory-hungry. It is recommended to have at least 1 GB of
-memory available when compiling Bitcoin Core. With 512MB of memory or less
-compilation will take much longer due to swap thrashing.
+memory available when compiling Bitcoin Core. However, if you are building on a
+system with less than 1 GB memory, the compiler can be instructed to conserve
+memory with additional CXXFLAGS:
+
+    ./configure CXXFLAGS="--param ggc-min-expand=1 --param ggc-min-heapsize=32768"
```

Comment:
nit. Looks like gcc specific. Maybe okay. But is there a clang equivalent or something that is supported by "all"/"most" compilers?